### PR TITLE
feat(suite): add coinjoin phases

### PR DESCRIPTION
**Pull request by dahaca**
## Description
 Added coinjoin phases

### CODE REVIEW

File: packages/suite/src/support/messages.ts [hunk 1]
```diff
@@ -7363,11 +7363,19 @@ export default defineMessages({
     },
     TR_COINJOIN_PHASE_1_MESSAGE: {
         id: 'TR_COINJOIN_PHASE_2_MESSAGE',
-        defaultMessage: 'Joining coins',
+        defaultMessage: 'Establishing connection',
     },
     TR_COINJOIN_PHASE_2_MESSAGE: {
         id: 'TR_COINJOIN_PHASE_3_MESSAGE',
-        defaultMessage: 'U poor now',
+        defaultMessage: 'Registering inputs',
```
Comment (szymonlesisz) [inline packages/suite/src/support/messages.ts]: TR_COINJOIN_PHASE_2_MESSAGE in id,

the message should be Registering outputs

File: packages/suite/src/support/messages.ts [hunk 2]
```diff
@@ -7363,11 +7363,19 @@ export default defineMessages({
...
+    },
+    TR_COINJOIN_PHASE_3_MESSAGE: {
+        id: 'TR_COINJOIN_PHASE_3_MESSAGE',
+        defaultMessage: 'Signing transactions',
+    },
+    TR_COINJOIN_PHASE_4_MESSAGE: {
+        id: 'TR_COINJOIN_PHASE_3_MESSAGE',
```
Comment (szymonlesisz) [inline packages/suite/src/support/messages.ts]: TR_COINJOIN_PHASE_4_MESSAGE in id

**Review by szymonlesisz — CHANGES_REQUESTED** [review]

File: packages/suite/src/support/messages.ts (hunk 1 above)
Comment (dahaca) [reply packages/suite/src/support/messages.ts]: 1a1e8b643dcab44d90e9c7ac142ad3ac5762308d

File: packages/suite/src/support/messages.ts (hunk 2 above)
Comment (dahaca) [reply packages/suite/src/support/messages.ts]: 1a1e8b643dcab44d90e9c7ac142ad3ac5762308d

**Review by szymonlesisz — APPROVED** [review]

**Pull request merged**